src/dorsal.js

```diff
@@ -160,7 +160,11 @@ DorsalCore.prototype._runPlugin = function(el, pluginName) {
  * @return {Array} registered plugin names
  */
 DorsalCore.prototype.registeredPlugins = function() {
-    return Object.keys(this.plugins);
```

Comment:
I like that. I was nervous about doing that, because there is a test that explicitly ensures that `plugins` is undefined on initialization, so I feared that something else would be reliant on that.